db/schema.ts

```diff
@@ -0,0 +1,22 @@
+import { integer, pgTable, serial, text, timestamp } from "drizzle-orm/pg-core";
+
+export const chatsTable = pgTable("chats", {
+  id: serial("id").primaryKey(),
+  name: text("name").notNull(),
+  createdAt: timestamp("created_at").notNull().defaultNow(),
+});
+
+export const messagesTable = pgTable("messages", {
+  id: serial("id").primaryKey(),
```

Comment:
Spend 5 minutes and learn about different possibities of selecting id, at least know the options.

Autoincrement number are usually frowned upon, either use UUIDv7 or later, or nanoid, or cuid or some random big int, there are many options.